source/renderer/app/i18n/locales/en-US.json

```diff
@@ -292,6 +292,7 @@
   "settings.stakePools.smash.descriptionIOHKLinkUrl": "https://iohk.io/en/blog/posts/2020/11/17/in-pools-we-trust/",
   "settings.stakePools.smash.descriptionLinkLabel": "Stakepool Metadata Aggregation Server (SMASH)",
   "settings.stakePools.smash.descriptionLinkUrl": "https://docs.cardano.org/en/latest/getting-started/stake-pool-operators/SMASH-metadata-management.html",
+  "settings.stakePools.smash.descriptionLoading": "Daedalus is synchronizing with the Cardano blockchain, and the process is currently <b>{syncPercentage}%</b> complete. This feature will become available once Daedalus is fully synchronized.",
```

Comment:
@daniloprates where did this come from? 🤔

Reply:
Not actually using it, will remove it.